theurgy.lic

```diff
@@ -1,186 +1,449 @@
-=begin
```

Comment:
I'd prefer to keep the header - the documentation on elanthipedia, while not great, is better than nothing

Reply:
Done.